### PR TITLE
EM - Auto Assignment - More granular settings

### PR DESCRIPTION
**Pull request by fsheik**
The schema is live so reviewing this PR against the [v1.0 $metadata](https://graph.microsoft.com/v1.0/$metadata).

Adding **removeAccessWhenTargetLeavesAllowedTargets** and **gracePeriodBeforeAccessRemoval** granular settings

### CODE REVIEW

File: changelog/Microsoft.IGAELM.json [hunk 1]
```diff
@@ -675,6 +675,24 @@
...
       "SubArea": "Governance"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "1dbbd2a3-caa8-49f3-bc3c-763ff8e6419a",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessPackageAutomaticRequestSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the **removeAccessWhenTargetLeavesAllowedTargets** and **gracePeriodBeforeAccessRemoval** properties to [accessPackageAutomaticRequestSettings](https://learn.microsoft.com/en-us/graph/api/resources/accessPackageAutomaticRequestSettings?view=graph-rest-1.0) resource type.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IGAELM.json:686]: This changelog is incorrect. Please use the autogenerated changelog or if you missed the window to download it, refer to the guidance for the structure of new properties.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @fsheik - The doc changes look good. The changelog changes not so. Please use the autogenerated changelog or if you missed the window to download it, refer to the guidance for the structure of new properties.

File: changelog/Microsoft.IGAELM.json (hunk 1 above)
Comment (fsheik) [reply changelog/Microsoft.IGAELM.json:686]: Hello @FaithOmbongi - I'm not sure what is missing in this changelog change, can you please point me relevant documentation?

**Review by FaithOmbongi — APPROVED** [review]
Hi @fsheik - I've left two suggestions for minor fixes and then approved your PR. Please resolve those before merging your PR.

Approved with comments.

**Pull request merged**